docs/filters.rst

```diff
@@ -45,6 +48,17 @@ e.g.
             exclude_keys: ['*_url', '_links', 'base', 'sender', 'owner', 'user']
         - dashes_to_underscores:
 
+
```

Comment:
You may want to push this example down after the next paragraph which explains what the first example is doing.